modules.d/01bootetc/boot-etc.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+. /lib/dracut-lib.sh
+
+bootetc=$(getarg bootetc=)
+if [ "${bootetc}x" == "x" ]; then
+    exit 0
+fi
+
+mount_boot bootetc
+if [ -d /boot/initrd-etc ] && [ ! -f /run/bootetc.done ]; then
+    info "bootetc: Updating initramfs etc from ${bootetc}/initrd-etc"
+    copytree /boot/initrd-etc /etc
+    touch /run/bootetc.done
+
+    if [ -z "$DRACUT_SYSTEMD" ]; then
+        systemctl try-restart dracut-cmdline.service
```

Comment:
This part is probably the most unfortunate.  I don't know all of what's going on in `dracut-cmdline.service` but it'll need to be prepared to be killed at arbitrary points.

Can't we have this bit run `Before=dracut-cmdline.service`?

Reply:
Nope, I looked at the boot-path. `dracut-cmdline.service` is run before Udev. we need udev to find the boot device.